src/Widget/SnippetWidget.php

```diff
@@ -21,10 +21,10 @@ public function __construct(
         string $target = Target::NOWHERE,
```

Comment:
Should we allow `string|string[]` here?

Reply:
I agree this should be changed, wonder if sticking to just `$targets` is better for consistency. Like, _always_ expect an array, without that many exceptions. So far, the only exception is in the `BaseWidget` because it's most widely used.

Reply:
Yeah, plural is better for consistency.. We'd just have to prevent BC-breaks if possible. If we can _say_ people should use the plural, but with a quick `if/else` we can prevent BC-breakage, that'd be best, imho. :-)

Reply:
Agreed! The latest changes should allow that :-)